python/mlc_chat/compiler/convert_weight.py

```diff
@@ -62,11 +63,18 @@ def _calc_total_params(model: nn.Module) -> int:
 
 def _convert_args(args: ConversionArgs) -> None:  # pylint: disable=too-many-locals
     # model config & quantization config
+    with Target.from_device(args.device) as target:
+        extern.enable(
+            target=target,
```

Comment:
nit: no need to `with` the target:

```suggestion
        extern.enable(
            target=Target.from_device(args.device),
```